app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,12 @@
+class SessionsController < ApplicationController
+  def create
```

Comment:
Nice work with this method

Reply:
Thanks! I figured it out entirely on my own.